tools/clang/test/HLSLFileCheck/hlsl/intrinsics/basic/abs1.hlsl

```diff
@@ -2,7 +2,7 @@
 
 // CHECK: main
 // After lowering, these would turn into multiple abs calls rather than a 4 x float
-// CHECK: call <4 x float> @"dx.hl.op..<4 x float> (i32, <4 x float>)"(i32 95,
+// CHECK: call <4 x float> @"dx.hl.op..<4 x float> (i32, <4 x float>)"(i32 96,
```

Comment:
Let's get rid of this test now that everything would break if something like this were really broken.  This test only generates noise now.